nautobot/core/templatetags/helpers.py

```diff
@@ -73,11 +74,16 @@ def hyperlinked_object(value, field="display"):
     """
     if value is None:
         return placeholder(value)
+    attributes = []
+    for k, v in kwargs.items():
+        if v is not None:
+            attributes.append(f'{k}="{v}"')
     display = getattr(value, field) if hasattr(value, field) else str(value)
     if hasattr(value, "get_absolute_url"):
+        attributes.insert(0, f'href="{value.get_absolute_url()}"')
         if hasattr(value, "description") and value.description:
-            return format_html('<a href="{}" title="{}">{}</a>', value.get_absolute_url(), value.description, display)
-        return format_html('<a href="{}">{}</a>', value.get_absolute_url(), display)
+            attributes.insert(1, f'title="{value.description}"')
+        return format_html("<a{}>{}</a>", mark_safe(" ".join(["", *attributes])), display)  # noqa: S308 # suspicious-mark-safe-usage Safe attributes
```

Comment:
Definitely let's not use `mark_safe` on an unknown set of attributes! I think it'd be better to use `format_html()` or `format_html_join()` (https://docs.djangoproject.com/en/3.2/ref/utils/#django.utils.html.format_html_join) to construct the individual entries in `attributes` and/or concatenate all of the `attributes` into a single safe string.

Reply:
@glennmatthews Refactored to use `format_html_join` which looks cleaner now too.